test/e2e_node/cos-init/default-kubelet-config.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kubelet.config.k8s.io/v1beta1
```

Comment:
Was this the default kubelet config settings for jenkins?

The cgroupDriver seems to be old..